dpd/src/types.rs

```diff
@@ -210,6 +215,12 @@ impl convert::From<DpdError> for dropshot::HttpError {
             DpdError::Faulted(e) => {
                 dropshot::HttpError::for_bad_request(None, e)
             }
+            DpdError::UnusableTransceiver => {
+                dropshot::HttpError::for_bad_request(
```

Comment:
I have no idea if it matters to callers, but this doesn't seem like a 400 error. It seems like a 503 or something like that? One way it could matter is if we act on this to say, retry a request in the future, e.g., if we expect the vendor info to _become_ available shortly.